dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/TestEnableDisableFeaturesTest.java

```diff
@@ -299,7 +299,7 @@ public void testEDF7() throws Exception {
     	currentServ = serverEDF6;
     	String testName = "testEDF7";
     	serverEDF6.startServer();
-    	Assert.assertNotNull("CWWKO0219I NOT FOUND",serverEDF6.waitForStringInLog("defaultHttpEndpoint-ssl",60000));
+    	Assert.assertNotNull("TCP Channel defaultHttpEndpoint-ssl has not started (CWWKO0219I not found)",serverEDF6.waitForStringInLog("CWWKO0219I.*defaultHttpEndpoint-ssl",60000));
```

Comment:
Is there a reason to keep this instead of changing to `waitForSecurityPrerequisites(serverEDF6, 60000);`?

Reply:
waitForSecurityPrerequisites checks for creation of LTPA keys but we don't need to as that was already created in the previous test case. They use the same server so the key only gets created onces, but we still need to check for the defaultHttpEndpoint-ssl message.